### PR TITLE
Rc1.1.17

### PR DESCRIPTION
**Pull request by coderidge**
Reporting for two new Apps - Active 10 and Days Off

### CODE REVIEW

File: src/TransformCore/PHE/AnalyticsBundle/Command/DailyDataExtractCommand.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ protected function configure()
                     array(
                         new InputOption(
                             'date',
-                            'day',
+                            'd',
```
Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Command/DailyDataExtractCommand.php]: Make sure this doesn't break any cron jobs

File: src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php [hunk 2]
```diff
@@ -378,6 +382,7 @@ public function getAllQuestionnairesInTimeSlice(\DateTime $start, \DateTime $end
 			return $questionnaire;
 		});
 
+		var_dump($results);
```
Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php]: can't leave this in the code when you want a merge

**Comment by coderidge** [comment]
Var_dump removed.  'd' was the original value in data extract, data extract works locally.

**Comment by brettminnie** [comment]
Merged, please rebase or merge into your next PR

**Pull request merged**